tests/artist-create.test.js

```diff
@@ -0,0 +1,35 @@
+// tests/artist-create.js
+const { expect } = require("chai");
+const request = require("supertest");
+const getDb = require("../src/services/db");
+const app = require("../src/app");
+
+describe("create artist", () => {
+  let db;
+  beforeEach(async () => (db = await getDb()));
+
+  afterEach(async () => {
+    await db.query("DELETE FROM Artist");
+    await db.close();
+  });
+
+  describe("/artist", () => {
+    describe("POST", () => {
+      it("creates a new artist in the database", async () => {
```

Comment:
Could a test be added to account for what would happen if the name or genre was missed off when creating the artist?